pmm-app/tests/verifyQAN_test.js

```diff
@@ -344,3 +346,100 @@ xScenario(
     await qanPage.verifyPagesAndCount(10);
   }
 );
+
+// TODO: Uncomment after new QAN will be merged
+xScenario('PMM-T133 - Verify user is able to add metric to the overview table', async (I, qanPage) => {
+  const testColumn = 'Bytes Sent';
+  const otherColumns = ['Query Count', 'Query Time', 'Load'];
+  qanPage.waitForNewQANPageLoaded();
+  I.click(qanPage.fields.addColumnNewQAN);
+  qanPage.addSpecificColumn(testColumn);
+  qanPage.verifyAddedColumn(testColumn);
+  for (i = 0; i < otherColumns.length; i++) {
+    qanPage.verifyAddedColumn(otherColumns[i]);
+  }
+});
+
+// TODO: Uncomment after new QAN will be merged
+xScenario('PMM-T132 - Verify user is able to change metric in the overview table', async (I, qanPage) => {
```

Comment:
Please search in existing Scenarios, T-132, T-133 are already covered, no need to duplicate.